### PR TITLE
HV-1515 Add new validation messages for locale zh_CN

### PR DESCRIPTION
**Pull request by quaff**
https://hibernate.atlassian.net/browse/HV-1515

### CODE REVIEW

**Comment by gsmet** [comment]
Jenkins, retest this please

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_zh_CN.properties [hunk 1]
```diff
@@ -1,27 +1,43 @@
...
+org.hibernate.validator.constraints.CodePointLength.message         = \u957f\u5ea6\u9700\u8981\u5728{min}\u548c{max}\u4e4b\u95f4
+org.hibernate.validator.constraints.LuhnCheck.message               = ${validatedValue}\u7684\u6821\u9a8c\u7801\u4e0d\u5408\u6cd5, Luhn\u6a2110\u6821\u9a8c\u548c\u4e0d\u5339\u914d
+org.hibernate.validator.constraints.Mod10Check.message              = ${validatedValue}\u7684\u6821\u9a8c\u7801\u4e0d\u5408\u6cd5, \u6a2110\u6821\u9a8c\u548c\u4e0d\u5339\u914d
+org.hibernate.validator.constraints.Mod11Check.message              = ${validatedValue}\u7684\u6821\u9a8c\u7801\u4e0d\u5408\u6cd5, \u6a2111\u6821\u9a8c\u548c\u4e0d\u5339\u914d
+org.hibernate.validator.constraints.ModCheck.message                = ${validatedValue}\u7684\u6821\u9a8c\u7801\u4e0d\u5408\u6cd5, ${modType}\u6821\u9a8c\u548c\u4e0d\u5339\u914d
+org.hibernate.validator.constraints.NotBlank.message                = \u4e0d\u80fd\u4e3a\u7a7a
+org.hibernate.validator.constraints.NotEmpty.message                = \u4e0d\u80fd\u4e3a\u7a7a
+org.hibernate.validator.constraints.ParametersScriptAssert.message  = script expression "{script}" didn't evaluate to true
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_zh_CN.properties]: Hi @quaff ,

Thanks for the update. Any reason why you didn't translate the line just above?

Comment (quaff) [reply engine/src/main/resources/org/hibernate/validator/ValidationMessages_zh_CN.properties]: Sorry, I forgot it, I have submitted another commit.

**Comment by gsmet** [comment]
Squashed and merged, thanks!

**Pull request closed without merging**